### DISCUSSION TITLE
Add boiler energy entities from telegram 0x2e and 0x3b

### DISCUSSION THREAD

**Discussion by kai-morich** [emsesp/EMS-ESP32#1701] [General]
I have a new hybrid heating system (Buderus WLW198 heat pump + GB212 gas boiler). 
The heatpump energy meter values are available with #1463 since the last version. Thanks!
This gas boiler provides similar values.
These are shown on the RC310 display and in the MyBuderus App retrieved over MX300 internet gateway. 
Please add those to the standard boiler entities.

Values are 4 bytes long and unit is 1/10 kWh.
For real offset, substract 1 from offset shown in custom entities screen shot, as custom entities are currently 3 bytes long.
They are calculated and broadcasted in ~ 10 minute intervals.
The values match the gas meter reporting m^3 with a calorific value of ~10.0.

    2024-04-18T18:59:38.272 T 253478: boiler(0x08) -B-> All(0x00), ?(0x2E), data: 00 00 1C CE 00 00 05 E8 00 00 00 18 00 00 00 02
    2024-04-18T18:59:38.449 T 253479: boiler(0x08) -B-> All(0x00), ?(0x3B), data: 00 00 1B D1 00 00 05 7F

<img width="542" alt="custom-entities" src="https://github.com/emsesp/EMS-ESP32/assets/25850372/855f34e5-0ab8-447b-8f4f-d566bc28edf6">

![screenshot_MyBuderus](https://github.com/emsesp/EMS-ESP32/assets/25850372/84639630-aa37-4550-923a-61221ae8dc11)

Note: the energy values calculated by EMS-ESP were factor 1.5 to high. I adjust the nominal power, but have no long term comparison yet.

**Comment by MichaelDvP** [comment]
Please add the support info. We need the product ids.

**Comment by kai-morich** [comment]
[emsesp_system_info.txt](https://github.com/emsesp/EMS-ESP32/files/15048174/emsesp_system_info.txt)

**Comment by kai-morich** [comment]
new entities available with 5ec68b85d2d26599149b571d2bc8fda26a1b1ad6, successfully tested with 3.7.0-dev.5

note: new naming scheme not consistent yet, e.g.
![grafik](https://github.com/emsesp/EMS-ESP32/assets/25850372/b815514d-2d14-4e30-ab66-48669df54c1d)
dhw in path and name. dhw and heat suffix could be omitted now

**Reply by proddy** [reply]
let's rename it

**Reply by proddy** [reply]
@MichaelDvP should we rename these below to remove the dhw at the end, since I believe they are all tagged with dhw/ ?

gasMeterWw (gasmeterdhw)
nrgWw (nrgdhw)
meterWw (meterdhw)
upTimeCompWw (uptimecompdhw)
nrgConsCompWw (nrgconscompdhw)
auxElecHeatNrgConsWW (auxelecheatnrgconsdhw)
nrgSuppWw (nrgsuppdhw)
wwAltOpPrioWw (altoppriodhw)
hpCircPumpWw (hpcircpumpdhw)

My only concern is some people may loose the history in HA as the entity_id will also change.

**Reply by MichaelDvP** [reply]
Yes, let's rename. In 3.6.5 they have suffix 'ww', 'dhw'  was already a rename. But we should add them to the compatibility check for 3.6 mqtt format. I'll add to next PR.
